Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -60,12 +60,18 @@ class SourceGenerator {
             localPackageGroup = addObject(PBXGroup(sourceTree: .sourceRoot, name: groupName))
             rootGroups.insert(localPackageGroup!)
         }
+        
+        let absolutePath = path.isAbsolute ? path : project.basePath + path.normalize()
```

Comment:
I believe PathKit automatically does this automatically already so if you add an absolute path to a relative path, it just evaluates to the absolute path